tracing/tests/span.rs

```diff
@@ -188,7 +188,7 @@ fn cloning_a_span_calls_clone_span() {
         .run_with_handle();
     with_default(subscriber, || {
         let span = span!(Level::TRACE, "foo");
-        let _span2 = span.clone();
+        let _span2 = span;
```

Comment:
I think this changes the test — this test is specifically that calling `clone` on the span will call into the subscriber's `clone_span` hook. Changing this to move the span means that we won't call `clone_span` on the subscriber, which _should_ result in the test failing.